docs/index.md

```diff
@@ -128,17 +128,14 @@ See the [Supported languages](/supported-languages#language-maturity-summary) do
 </div>
 -->
 
-<h3>June 2025 release notes summary</h3>
+<h3>July 2025 release notes summary</h3>
 <!-- 5-7 bullets across the product suite -->
 
-- You can now **customize PR and MR comments** to provide additional context to the comments generated by Semgrep.
-- The **Finding details** page has been redesigned and unified across all Semgrep products.
-- Support for **PHP** reachability is now in **public beta**, which means that Semgrep offers 98% coverage for **Critical** severity issues, plus some coverage for **High** severity issues.
-- **C#**:
-  - [Dependency Paths](/semgrep-supply-chain/dependency-search#view-the-dependency-path) for C# projects using NuGet are now in **public beta**.
-  - Semgrep can scan NuGet codebases without the need for a lockfile. This feature is in **public beta**.
-- **Memories** can now be scoped to a rule's vulnerability class, which are the same groupings that exist on the policies page.
-- You can now create **memories for generic secrets**, allowing you to create and apply custom rules for secret detection through Assistant.
+- Support for running Semgrep natively on Windows is now in **public beta**. This applies to running Semgrep through the CLI and an IDE such as Cursor, VS Code, and IntelliJ.
+- Supply Chain support for PHP is now **generally available (GA)**.
```

Comment:
Do we need to specify that it’s reachability analysis that is GA, not "vanilla" PHP support?